File: Python/25 March 2025 Homework/Task1.js
// Task 1
// Написать свой класс для хранения информации о человеке: фамилия, имя, отчество, дата рождения

class MenInfo {
    constructor(name, surname, patronymic, year, height) {
        this.name = name;
        this.surname = surname;
        this.patronymic = patronymic;
        this.year = year;
        this.height = height;
    }

    calculateAge() {
        const currentYear = new Date().getFullYear();
        return currentYear - this.year;
    }
}

const mi = new MenInfo("Андрей", "Пшеничный", "Михайлович", 1998, 171);

console.log(mi.name);
console.log(mi.surname);
console.log(mi.patronymic);
console.log(mi.year);
console.log(mi.calculateAge());

File: Python/25 March 2025 Homework/Task1.py
# Task 1
# Написать свой класс для хранения информации о человеке: фамилия, имя, отчество, дата рождения

from datetime import datetime

class MenInfo:
    def __init__(self, name, surname, patronymic, year, height):
        self.name = name
        self.surname = surname
        self.patronymic = patronymic
        self.year = year
        self.height = height
    
    def calculate_age(self):
        current_year = datetime.now().year
        return current_year - self.year


mi = MenInfo("Андрей", "Пшеничный", "Михайлович", 1998, 171)

print(mi.name)
print(mi.surname)
print(mi.patronymic)
print(mi.year)
print(mi.calculate_age())